storage/local/interface.go

```diff
@@ -34,6 +34,9 @@ type Storage interface {
 	// from the provided Sample as those labels are considered equivalent to
 	// a label not present at all.
 	Append(*model.Sample)
+	// Throttled returns true if the Storage has too many chunks in memory
+	// already or has too many chunks waiting for persistence.
+	Throttled() bool
```

Comment:
`Throttle()` sounds more like a method to throttle the storage, not like a question. I was thinking about the inverted logic, but then it always sounds like the storage is incapable of ingesting more, which is not true. It just wants fewer samples so that it can recover from the overload.

How about `NeedsThrottling()`?